test/e2e/customconfigs/skipjobswithoutqueuename_test.go

```diff
@@ -343,7 +343,7 @@ var _ = ginkgo.Describe("ManageJobsWithoutQueueName", ginkgo.Ordered, func() {
 				pods := &corev1.PodList{}
 				gomega.Expect(k8sClient.Delete(ctx, testDeploy)).Should(gomega.Succeed())
 				gomega.Eventually(func(g gomega.Gomega) {
-					g.Expect(k8sClient.List(ctx, pods, client.InNamespace(ns.Namespace),
+					g.Expect(k8sClient.List(ctx, pods, client.InNamespace(ns.Name),
```

Comment:
```suggestion
					g.Expect(k8sClient.List(ctx, pods, client.InNamespace("kube-system"),
```
? I think all the checks should target the same namespace within each it?

Reply:
I think this passes only "by accident" because we assert the set to be empty, but checking "kube-system" is the intention of the test.

Reply:
Yeah, you right, missed here. Thank you!